server/routes/routes.js

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Having a alot of commented code is not proper code convention. If you are not using the code then you should simple delete it.